x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/preview_rules_route.ts

```diff
@@ -92,14 +91,7 @@ export const previewRulesRoute = async (
         const siemClient = (await context.securitySolution).getAppClient();
 
         let invocationCount = request.body.invocationCount;
-        if (
-          ![
-            RULE_PREVIEW_INVOCATION_COUNT.HOUR,
-            RULE_PREVIEW_INVOCATION_COUNT.DAY,
-            RULE_PREVIEW_INVOCATION_COUNT.WEEK,
-            RULE_PREVIEW_INVOCATION_COUNT.MONTH,
-          ].includes(invocationCount)
-        ) {
+        if (invocationCount < 1) {
```

Comment:
Could we also put that [200 invocation limit](https://github.com/elastic/kibana/pull/137102/files#diff-c4417134cf89472f7a8a389079dc0ab4a9ed7de62c595961351cb734e7ef4060R29) here as well?

Reply:
Not really. The 200 is the number of invocations that we think is reasonable. If user sets the timeframe and the rule interval in a way that invocation count goes above 200 we will still accept it, but will warn the user that the preview might timeout or take a long time.